### DISCUSSION TITLE
Globally change default document.body for Portals

### DISCUSSION THREAD

**Discussion by SevereOverfl0w** [mantinedev/mantine#5553] [Feature requests]
I'm mounting components into iframes to isolate them from the main page in a sort of "storybook-like" environment.  Unfortunately for the Combobox (which uses Portal) this means that it finds the top page's document and renders outside of the iframe.  Instead I'd like it to use the `body` inside of the iframe.

It's tedious to specify this for every use of Combobox.  Instead it would be better if I could specify it once as the default in `MantineProvider` (similarly to how it already takes a `getRootElement`) and that's what `Portal` uses if it isn't given a specific target.

It would be really handy to override document.body for Portals globally in MantineProvider.

**Comment by rtivital** [comment]
Assign default props to Portal component – https://mantine.dev/theming/default-props/

**Comment by SevereOverfl0w** [comment]
Right now, Portal doesn't support an alternative document.body, it will always use document.body.  So there's two requirements here:

1. That components referencing `document` have the option to take that as a param
2. That I can set it globally (presumably I can do that using this defaultProps solution, though!)
